### DISCUSSION TITLE
COMO SUBIR ARCHIVO DE PRACTICA HTML AL FORMULARIO

### DISCUSSION THREAD

**Discussion by RaulGuereca** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#271] [Q&A]
Tengo mi archivo de programa de HTML del cliente y el pastelero pero como lo subo en el formulario que nos piden. me podrian decir que es lo que tengo que hacer.  gracias,

**Comment by hackit92** [comment]
Que tal Raul, crea un nuevo repositorio en tu Github destinado a esta práctica en modo público, carga ahí todos los archivos que contenga tu práctica (html, imagenes, css) y ese será el enlace que pondrás en el formulario, espero haberte sido de ayuda.

**Comment by JuanSalvador73** [comment]
Al crear un nuevo repositorio, subes tu archivo HTML, yo lo subo con todo y folder de mi máquina local. Para subir archivos es en donde dice "add files" al crear el repositorio.

**Reply by RodolfoMorquecho** [reply]
disculpa, ya ubi todos los archivos/imagenes pero tengo la duda si se debe de poner el link de la pagina creada tambien?

**Reply by MarcosSilvaG** [reply]
se debe de poner si o sí el link del repositorio, poner la página online era un extra :D saludos
